OpenRA.Launcher/Properties/launchSettings.json

```diff
@@ -15,6 +15,10 @@
     "Tiberian Sun": {
       "commandName": "Project",
       "commandLineArgs": "Engine.EngineDir=\"..\" Game.Mod=ts"
+    },
+    "Red Alert - Test Map": {
```

Comment:
Was this meant to be committed, or did you use it for local testing?

Reply:
Yes only for testing, I'll remove it

Reply:
Have now removed this as it was only for testing purposes.